force-app/main/default/classes/DMLOperations.cls

```diff
@@ -158,15 +204,24 @@ public with sharing class DMLOperations {
 	 * @return The Account that was upserted.
 	 */
 	public static Account upsertAccount(String accountName) {
+        Account accToUpdate;
 		// Query for the Account
 		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
 
 		// if the Account exists, update the description
-
-		// otherwise, create a new Account and set the description
-
+        if (!accounts.isEmpty()) {
+            accToUpdate = accounts[0];
+            accToUpdate.Description = 'Updated Account';
+        } else {
+            // otherwise, create a new Account and set the description
+            Account newAccount = new Account();
```

Comment:
You can also set acctToUpdate = new Account(); rather than creating a new variable